backend/apps/auth/src/routes/auth.py

```
@@ -0,0 +1,32 @@
+from fastapi import APIRouter, Depends, HTTPException
+from fastapi.security import HTTPBearer
+
+from src.utils.jwt import verify_jwt_token
+
+router = APIRouter()
+security = HTTPBearer()
+
+@router.post("/verify")
+def verify_token(credentials = Depends(security)):
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)